Emergence/Contexts/App Setup/AppDelegate.swift

```diff
@@ -15,10 +26,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // When someone clicks on a top shelf item
         guard let appVC = window?.rootViewController as? AppViewController else { return false }
-        let showID = url.pathComponents?.last
+        guard let showID = url.pathComponents?.last else { return false }
         appVC.openShowWithID(showID)
 
+        ARAnalytics.event("tapped featured shows", withProperties: ["partner_show_id": showID])
         return true
     }
-}
 
+    func applicationDidBecomeActive(application: UIApplication) {
+        ARAnalytics.startTimingEvent("app session time")
```

Comment:
Could be put in constants. Slippery slope I know, but only because the same key has to be in two places.